### ISSUE TITLE
Error: node-canvas was built without SVG support

### ISSUE THREAD

**Issue by gee1k** [Automattic/node-canvas#1484]
## Issue or Feature

Error reporting when loading SVG with loadImage
## Steps to Reproduce

```js
var Canvas = require('canvas');
var canvas = Canvas.createCanvas(200, 200);
var ctx = canvas.getContext('2d');
// etc.
var iconUrl = 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxNi4zOCAxNi4zOCI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOiMyMmNiZmY7c3Ryb2tlOiMwMDAwMzc7c3Ryb2tlLW1pdGVybGltaXQ6MTA7c3Ryb2tlLXdpZHRoOjAuMzhweDt9LmNscy0ye2ZpbGw6I2ZmZjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPui1hOa6kCAyNTwvdGl0bGU+PGcgaWQ9IuWbvuWxgl8yIiBkYXRhLW5hbWU9IuWbvuWxgiAyIj48ZyBpZD0i5Zu+5bGCXzEtMiIgZGF0YS1uYW1lPSLlm77lsYIgMSI+PGNpcmNsZSBjbGFzcz0iY2xzLTEiIGN4PSI4LjE5IiBjeT0iOC4xOSIgcj0iOCIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTEyLjY0LDExLjhWNy40OEgxMC44OVYyLjc3YS4yOS4yOSwwLDAsMC0uMjktLjI5SDUuNzhhLjI5LjI5LDAsMCwwLS4yOS4yOVY3LjQ4SDMuNzRWMTEuOEgzdi41OEgxMy40MlYxMS44Wm0tNC04LjExSDkuOTRWNUg4LjY3Wm0wLDEuODFIOS45NFY2Ljc3SDguNjdabTAsMS44SDkuOTRWOC41N0g4LjY3Wk04LjI0LDguODNsLjQ0LjksMSwuMTUtLjcxLjcuMTYsMS0uODgtLjQ3LS44OS40Ny4xNy0xLS43Mi0uNywxLS4xNVpNNi40NCwzLjY5SDcuNzFWNUg2LjQ0Wm0wLDEuODFINy43MVY2Ljc3SDYuNDRabTAsMS44SDcuNzFWOC41N0g2LjQ0WiIvPjwvZz48L2c+PC9zdmc+'
Canvas.loadImage(iconUrl).then(img => {
      ctx.drawImage(img, 100, 100, 23, 23)
    })
```

## Your Environment
* Version of node-canvas (output of `npm list canvas` or `yarn list canvas`):
`canvas@2.6.0`
* Environment (e.g. node 4.2.0 on Mac OS X 10.8):
`Mac OS X 10.15`

**Comment by NoTalk-ly** [comment]
is anybody give attention to this problem?
this alse happened in canvas@2.6.1, I use to loaImage which type is SVG.

**Comment by NoTalk-ly** [comment]
https://github.com/Automattic/node-canvas/issues/1513
the mentioned method fixed this problem.

**Comment by zbjornson** [comment]
From the last comment, sounds like this was from building from source without librsvg installed.

**Comment by taxProper-bryan** [comment]
I came across this error still (Sep 2023), so documenting what I did to address it. I am using `canvas=2.11.2` and `node=20.3.1`. I initially tried installing `librsvg` and reinstalling node-canvas with no luck. What seemed to work for me was:

1. Uninstalling `canvas`: `npm uninstall canvas`
2. Installing `librsvg`
3. Rebuilding canvas: `npm rebuild --build-from-source=canvas`
4. Reinstall canvas: `npm install canvas`

**Comment by turbodog03** [comment]
Encountered this error this error on Windows with `canvas=2.11.2` and `node=20.10.0`, `node_gyp=10.0.1` yesterday. Tried almost every solution mentioned, and have a hard time trying to install librsvg. Then I found they have pre-built binaries in [the release](https://github.com/Automattic/node-canvas/releases). 

Here are the steps I took:
1. install canvas `npm install canvas` (if have it uninstalled)
2. Navigate to the `node_modules > canvas > build` folder.
3. Download correct version of pre-built release, unzip it, and paste it into the `build` folder ( rewrite the original `Release` folder if exists )

This approach resolved the issue for me  :)
